src/components/SearchFilters/SearchFilters.tsx

```diff
@@ -7,10 +7,21 @@ import { P } from "../Typography";
 import SearchFilterCheckbox from "./SearchFilterCheckbox";
 
 const SearchFilters: FC<UseSearchFiltersReturnType> = (props) => {
-  const { keyStageFilters, subjectFilters } = props;
-
+  const { keyStageFilters, subjectFilters, searchTypeFilters } = props;
   return (
     <>
+      <P $mb={16}>Type</P>
+      <Flex $mb={36} $flexWrap={"wrap"}>
+        {searchTypeFilters.map((type) => (
```

Comment:
hmm thinking "type" is not a good name, as my be confused with the type. what is it a filter for?

Reply:
changed type to contentType